backend/dal/Services/Admin/Concrete/ProjectService.cs

```diff
@@ -136,23 +168,12 @@ public void Add(IEnumerable<Project> projects)
 
             projects.ForEach((project) =>
             {
-                if (project == null) throw new ArgumentNullException();
-
-                project.Workflow = null;
-                project.Status = null;
-                project.Agency = null;
-                project.TierLevel = null;
-                project.Risk = null;
+                if (project == null) return;
 
-                project.Responses.ForEach(r =>
-                {
-                    r.Agency = null;
-                });
-
-                project.Tasks.ForEach(t =>
+                if (String.IsNullOrWhiteSpace(project.ProjectNumber))
                 {
-                    t.Task = null;
-                });
+                    project.ProjectNumber = $"TEMP-{DateTime.UtcNow.Ticks:00000}";
```

Comment:
Are temp projects viewable on the FE? If so does it display ok?

Reply:
They are viewable, as they are just projects that haven't been submitted.